ocd-2885.sql

```diff
@@ -0,0 +1,3 @@
+DROP SCHEMA IF EXISTS ff4j CASCADE;
+
+\i dev/ff4j-schema.sql
```

Comment:
Just tried running my v-next locally to get the schema change but the filename you have here is not the file that has the schema. Should be openchpl_ff4j.sql